Formatter/SubjectFormatter.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Jstack\Newyse\Formatter;
+
+class SubjectFormatter
+{
+    /**
+     * @param $subjects
+     * @return array
+     */
+    public function getSubjectQuantities($subjects)
```

Comment:
Since you wrote a formatter, I'd also call the public methods accordingly. E.g. this could be `formatSubjectQuantities`